added index.html & CSS

rancescolon opened the pull request:
HTML and CSS

rancescolon committed:
added index.html & CSS

Eric-Butcher commented on evergreen/index.html:
Change Ever Green to Evergreen.

Also I would eliminate the sections about the clubs other projects. For now lets just focus on the evergreen greenhouse. Also for the contact us, I would add the discord invite link: https://discord.com/invite/QMHjt65z7a.

Eric-Butcher commented on evergreen/styles.css:
I don't know CSS enough to authoritatively approve this on my own. But it looks ok so far to my eyes.

WeirdAlex03 reviewed (commented):
I'm not great on my CSS either but lgtm. With the "Project Evergreen" changes I'll signoff

rancescolon committed:
added changes to HTML and CSS

made Project-Evergreen consistent, removed other projects and changed email link to discord

WeirdAlex03 commented on evergreen/index.html:
```diff
@@ -0,0 +1,31 @@
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Project-Evergreen</title>
```
Just "`Project Evergreen`" - space, not dash

rancescolon committed:
Project-Evergreen -> Project Evergreen

WeirdAlex03 reviewed (approved):
:shipit:
